gtsam/hybrid/HybridBayesNet.cpp

```diff
@@ -20,21 +20,25 @@
 
 namespace gtsam {
 
+/* ************************************************************************* */
 GaussianMixture::shared_ptr HybridBayesNet::atGaussian(size_t i) const {
   return boost::dynamic_pointer_cast<GaussianMixture>(factors_.at(i));
 }
 
+/* ************************************************************************* */
 void HybridBayesNet::setGaussian(size_t i,
-                                 const GaussianMixture::shared_ptr& gaussian) {
+                                 const GaussianMixture::shared_ptr &gaussian) {
```

Comment:
Nope, no non-const monkey business :-)

Reply:
Good call. I realized this method is unnecessary now.